src/applications/app.helpers.js

```diff
@@ -66,7 +66,7 @@ export function withoutLoadErrors(app) {
 }
 
 export function toName(app) {
-  return app.appOrParcelName || app.name;
```

Comment:
Removed unneeded code - `appOrParcelName` is only ever added to error objects, not to actual apps or parcels.